ads_plugin.py

```diff
@@ -319,7 +319,10 @@ def on_apply_model_button(self, event):
         # Get the path of the selected model
         if any(selected_model in models for models in ads_utils.get_existing_models_list()):
             model_path = os.path.dirname(AxonDeepSeg.__file__)
```

Comment:
Instead of using mode_path as a dummy variable here that gets rewritten, we should fix it and use a different name because it was confusing for me to understand what was going on at first site. I'll make this change.

Reply:
Resolved in 971a46c